index.html

```diff
@@ -17,11 +18,24 @@
 <body>
 
  <!-- navigation code here -->
+  <nav class="nav">
+    <h1>Space Walkers</h1>
+    <div class="navLinks">
+      <a href="#">Home</a>
+      <a href="#">About Us</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+      <a href="#">Sign In</a>
+    </div>
+  </nav>
 
   <div class="container home">
 
     <header class="intro">
        <!-- image code here -->
+       <div>
+         <img class="headerPic" src="/img/main-header.png" alt="Earth from space.">
```

Comment:
what is different about this src tag than your other src tags on the page?

Reply:
Well, every time I work with images, I prefer to put them inside a div because it's easier to get them to do what you want them to do when just telling the img or the class without something wrapped around it. Honestly, I ran out of time to get to the other three images.